tests/topotests/bgp_multi_vrf_topo2/test_bgp_multi_vrf_topo2.py

```diff
@@ -2011,6 +2028,1782 @@ def test_vrf_route_leaking_next_hop_interface_flapping_p1(request):
+                                        "dest_link": {
+                                            "blue2-link1": {
+                                                "keepalivetimer": KEEPALIVETIMER,
+                                                "holddowntimer": HOLDDOWNTIMER,
+                                            }
+                                        }
+                                    }
+                                }
+                            }
+                        },
+                    },
+                },
+                {
+                    "local_as": "800",
+                    "vrf": "BLUE_B",
+                    "address_family": {
+                        "ipv4": {
+                            "unicast": {
+                                "neighbor": {
+                                    "r3": {
+                                        "dest_link": {
+                                            "blue2-link2": {
+                                                "keepalivetimer": KEEPALIVETIMER,
+                                                "holddowntimer": HOLDDOWNTIMER,
+                                            }
+                                        }
+                                    }
+                                }
+                            }
+                        },
+                        "ipv6": {
+                            "unicast": {
+                                "neighbor": {
+                                    "r3": {
+                                        "dest_link": {
+                                            "blue2-link2": {
+                                                "keepalivetimer": KEEPALIVETIMER,
+                                                "holddowntimer": HOLDDOWNTIMER,
+                                            }
+                                        }
+                                    }
+                                }
+                            }
+                        },
+                    },
+                },
+            ]
+        },
+    }
+
+    result = create_router_bgp(tgen, topo, input_dict_4)
+    assert result is True, "Testcase {} :Failed \n Error: {}".format(tc_name, result)
+
+    for addr_type in ADDR_TYPES:
+        clear_bgp(tgen, addr_type, "r3", vrf=["RED_A", "RED_B", "BLUE_A", "BLUE_B"])
+
+        clear_bgp(tgen, addr_type, "red2", vrf=["RED_A", "RED_B"])
+
+        clear_bgp(tgen, addr_type, "blue2", vrf=["BLUE_A", "BLUE_B"])
+
+    step("Delete vrfs RED_A and BLUE_A from R3")
+
+    input_dict = {
+        "r3": {
+            "vrfs": [
+                {"name": "RED_A", "id": "1", "delete": True},
+                {"name": "BLUE_A", "id": "3", "delete": True},
+            ]
+        }
+    }
+
+    result = create_vrf_cfg(tgen, input_dict)
+    assert result is True, "Testcase {} :Failed \n Error {}".format(tc_name, result)
+
+    step("Waiting for holddowntimer+1..")
```

Comment:
Can you fix this everywhere in this file as you did it already in other places here?

Reply:
Hi Donatas, 
I have fixed it everywhere. Thank you.